code/modules/antagonists/heretic/knowledge/void_lore.dm

```diff
@@ -275,3 +316,33 @@
 	if(storm)
 		storm.end()
 		QDEL_NULL(storm)
+
+///Few checks to determine if we can deflect bullets
+/datum/heretic_knowledge/ultimate/void_final/proc/can_deflect(mob/living/ascended_heretic)
+	if(!ascended_heretic.combat_mode)
+		return FALSE
+	if(ascended_heretic.incapacitated(IGNORE_GRAB)) //NO STUN
+		return FALSE
+	if(!(ascended_heretic.mobility_flags & MOBILITY_USE)) //NO UNABLE TO USE
+		return FALSE
+	var/datum/dna/dna = ascended_heretic.has_dna()
+	if(dna?.check_mutation(/datum/mutation/human/hulk)) //NO HULK
+		return FALSE
+	if(!isturf(ascended_heretic.loc)) //NO MOTHERFLIPPIN MECHS!
+		return FALSE
+	return TRUE
```

Comment:
I know this is copypasted because you copy pasted the comments, at least remove those or provide your own explanation as to why these are forbidden

To that end do we really need to block hulk from this? Does the void really care if you're a big green guy or not?